spec/dummy/app/models/authentication.rb

```diff
@@ -0,0 +1,7 @@
+class Authentication < ApplicationRecord
+  include PasswordConcern
```

Comment:
Just use `ManageIQ::Password::PasswordMixin`, then no need to duplicate it in this project.